libs/core/langchain_core/language_models/llms.py

```diff
@@ -119,7 +119,7 @@ def _before_sleep(retry_state: RetryCallState) -> None:
     # 4 seconds, then up to 10 seconds, then 10 seconds afterwards
     retry_instance: retry_base = retry_if_exception_type(error_types[0])
     for error in error_types[1:]:
-        retry_instance = retry_instance | retry_if_exception_type(error)
+        retry_instance |= retry_if_exception_type(error)
```

Comment:
Is this an auto fix as well?

Reply:
Yes, it’s an auto-fix

Reply:
An "unsafe" one since the first syntax creates a new object each time whereas `|=` may mutate the initial object.
Here it's OK to mutate `retry_instance`.